package.json

```diff
@@ -34,7 +35,9 @@
     "dep-graph": "nx dep-graph",
     "help": "nx help",
     "storybook:island-ui": "nx run island-ui-storybook:storybook",
-    "ts-node": "node -r ./scripts/ts-node"
+    "ts-node": "ts-node -r tsconfig-paths/register -O '{\"module\":\"commonjs\",\"esModuleInterop\":true}'",
```

Comment:
I actually want the `scripts/ts-node` file so I can later require it from js files (like `next.config.js`) and then import shaerd typescript code.

Did you have issues with it?

Reply:
Oh my bad, it wasn't used so I thought I could overwrite it. 

Yes I couldn't pass down the arguments for ts-node e.g. `ts-node -P tsconfig.json file.ts`, probably because it's using `node` instead of `ts-node`.

I can reintroduce another `ts-node-whatever` script specifically for the workspace usage

What do you think?

Reply:
Ah, I see. Makes sense.

Just undelete it for now. No need to create another script in package.json.

Reply:
Not sure to understand your comment _Just undelete it for now_. Did you mean _delete_?

Reply:
Yeah, I meant to restore the old scripts/ts-node.js file and leave it even though it's not used anymore in `yarn ts-node`.